core/src/main/java/com/netflix/conductor/core/execution/DeciderService.java

```diff
@@ -625,8 +625,9 @@ void checkWorkflowTimeout(WorkflowModel workflow) {
 
         String reason =
                 String.format(
-                        "Workflow timed out after %d seconds. Timeout configured as %d seconds. "
+                        "Workflow %s timed out after %d seconds. Timeout configured as %d seconds. "
```

Comment:
This failure reason is added to the workflow object which already contains the id. Is this change required?

Reply:
I don't see this variable being used to set the workflow object. It's only used in the log statement [line 637](https://github.com/Netflix/conductor/blob/7d77df658df6bd4c4aaf7e11cbfcb717c79c90d3/core/src/main/java/com/netflix/conductor/core/execution/DeciderService.java#L637)

Reply:
line 644 here propagates the reason to the workflow object

Reply:
Ahh .. I can update the log statement separately and leave the reason as it was before